php/laravel-laravel-s/config/laravels.php

```diff
@@ -0,0 +1,322 @@
+    |
+    */
+
+    'register_providers' => [],
+
+    /*
+    |--------------------------------------------------------------------------
+    | Cleaners
+    |--------------------------------------------------------------------------
+    |
+    | The list of cleaners for each request is used to clean up some residual
+    | global variables, singleton objects, and static properties to avoid
+    | data pollution between requests.
+    |
+    | https://github.com/hhxsv5/laravel-s/blob/master/Settings.md#cleaners
+    |
+    */
+
+    'cleaners' => [],
+
+    /*
+    |--------------------------------------------------------------------------
+    | Destroy Controllers
+    |--------------------------------------------------------------------------
+    |
+    | Automatically destroy the controllers after each request to solve
+    | the problem of the singleton controllers.
+    |
+    | https://github.com/hhxsv5/laravel-s/blob/master/KnownIssues.md#singleton-controller
+    |
+    */
+
+    'destroy_controllers' => [
+        'enable'        => false,
+        'excluded_list' => [],
+    ],
+
+    /*
+    |--------------------------------------------------------------------------
+    | Swoole Settings
+    |--------------------------------------------------------------------------
+    |
+    | Swoole's original configuration items.
+    |
+    | More settings
+    | Chinese https://wiki.swoole.com/#/server/setting
+    | English https://www.swoole.co.uk/docs/modules/swoole-server/configuration
+    |
+    */
+
+    'swoole' => [
+        'daemonize'          => env('LARAVELS_DAEMONIZE', false),
+        'dispatch_mode'      => 2,
+        'reactor_num'        => env('LARAVELS_REACTOR_NUM', function_exists('swoole_cpu_num') ? swoole_cpu_num() : 8),
+        'worker_num'         => env('LARAVELS_WORKER_NUM', function_exists('swoole_cpu_num') ? swoole_cpu_num() : 8),
+        //'task_worker_num'    => env('LARAVELS_TASK_WORKER_NUM', function_exists('swoole_cpu_num') ? swoole_cpu_num() * 2 : 8),
+        'task_ipc_mode'      => 1,
+        'task_max_request'   => env('LARAVELS_TASK_MAX_REQUEST', 100000),
+        'task_tmpdir'        => @is_writable('/dev/shm/') ? '/dev/shm' : '/tmp',
+        'max_request'        => env('LARAVELS_MAX_REQUEST', 100000),
+        'open_tcp_nodelay'   => true,
+        'tcp_fastopen'       => true,
+        'pid_file'           => storage_path('laravels.pid'),
+        'log_level'          => 4,
+        'log_file'           => storage_path(sprintf('logs/swoole-%s.log', date('Y-m'))),
+        'buffer_output_size' => 2 * 1024 * 1024,
+        'socket_buffer_size' => 8 * 1024 * 1024,
+        'package_max_length' => 4 * 1024 * 1024,
+        'reload_async'       => true,
+        'max_wait_time'      => 60,
+        'enable_reuse_port'  => true,
+        'enable_coroutine'   => false,
+        'upload_tmp_dir'     => @is_writable('/dev/shm/') ? '/dev/shm' : '/tmp',
+        'http_compression'   => false,
+    ],
```

Comment:
could you use ?
```suggestion
    'swoole' => [
          'worker_num'       => swoole_cpu_num() * 2,
            'enable_coroutine' => false,
            'log_file' => '/dev/null',
            'log_level' => SWOOLE_LOG_ERROR,
    ],
```
then idea is to unify `swoole` config for all `swoole` based **frameworks**